.github/workflows/stale.yml

```diff
@@ -0,0 +1,12 @@
+name: Stale PRs
+
+on:
+  schedule:
+    # Everyday at 2am
+    - cron: '0 2 * * *'
```

Comment:
Do you need to specify that this should only happen on the master branch?  (It seems like maybe not, but I'm not sure...)

Reply:
No idea. How can we know?

Reply:
I think it is reasonable to expect that schedules will only run for the default branch.

Reply:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule
>Scheduled workflows run on the latest commit on the default or base branch.